### ISSUE TITLE
Switch GPIO by http command

### ISSUE THREAD

**Issue by wiredcharlie** [sle118/squeezelite-esp32#220]
New feature to allow switching of GPIOs by sending an instruction over http.

[https://forums.slimdevices.com/forum/user-forums/3rd-party-hardware/109412-announce-squeezelite-esp32-dedicated-thread/page120#post1628955](url)

**Comment by wiredcharlie** [comment]
Is this something I could test, please?

**Comment by sle118** [comment]
@wiredcharlie unfortunately for the short term, we are restricting the code changes happening to the current branch. I recommend you have a read at this: https://github.com/sle118/squeezelite-esp32/discussions/342

I am committing my spare time to re-writing a good portion of the underlying system to make it more robust and scalable, but this takes time and although I have made good progress since I started last fall, the code is still not ready to be published. This is done as a fun project that consumes spare time, which is limited considering all the other commitments I have in my personal life.

When the new branch is published, there will be a period of stabilization and then only we will open the door for new features.
